erdjs/src/smartcontracts/typesystem/typeMapper.spec.ts

```diff
@@ -42,6 +43,11 @@ describe("test mapper", () => {
         testMapping("OptionalResult<Address>", OptionalType, [new AddressType()]);
     });
 
+    it("should map tuples", () => {
+        testMapping("tuple2<u32,bytes>", TupleType, [new U32Type(), new BytesType()]);
+        testMapping("tuple2<Address,BigUint>", TupleType, [new AddressType(), new BigUIntType()]);
```

Comment:
Ideas for more tests here: `tuple4` and `tuple2<tuple3<u32, bytes, u64>, Address>`.

Reply:
Added one more test for tuple3 and added a comment to change the serializer logic in a different PR.